keps/NNNN-kep-template/README.md

```diff
@@ -270,6 +266,55 @@ when drafting this test plan.
 [testing-guidelines]: https://git.k8s.io/community/contributors/devel/sig-testing/testing.md
 -->
 
+[ ] I/we understand the owners of the involved components may require updating
+existing tests to make this code solid enough prior committing changes necessary
+to implement this enhancement.
+
+<!--
+Based on reviewers feedback describe what additional tests need to be added prior
+implementing this enhancement to ensure the enhancements have also solid foundations.
+-->
+
+##### Unit tests
+
+<!--
+In principle every added code should be unit tested. However, the exact tests are hard
+to answer ahead of implementation. As a result, this section should only be filled when
+targeting Beta to ensure that something wasn't missed during Alpha implementation.
```

Comment:
Do we want any updates to this for GA or just beta?

Reply:
it should be no-op for GA, but it doesn't hurt to add it - added.